admin/js/wizard.js

```diff
@@ -19,6 +19,62 @@ function buildFormData (formData, data, parentKey) {
     formData.append(parentKey, value);
   }
 };
+function installPlugin(event) {
```

Comment:
This is a bit confusing having 3 functions with the work install in them. I see why they are all needed, but maybe we can name them better. 

The one for the js event could be `onInstallClick` to clearly indicate it is a js click event.

We could then name the others `installByUrl` and `installBySlug`.